docs/metrics.md

```diff
@@ -0,0 +1,25 @@
+
+---
+title: Metrics
+weight: 3200
+indent: true
+---
+
+# Metrics 
+
+CSI deploys a sidecar container that is responsible for collecting metrics.
+
+## Liveness
+
+Liveness metrics are intended to collected by prometheus but can be acceses through a GET request to a specific node ip.
+
+for example `curl -X get http://[node ip]:[liveness-port][liveness-path] 2>/dev/null | grep csi`
+
+the expected out put should
+
+```bash
+[root@worker2 /]# curl -X GET http://10.109.65.142:8080/metrics 2>/dev/null | grep csi
+# HELP csi_liveness Liveness Probe
```

Comment:
The pod IP you make the request to would indicate which which container you are making the request too. This is not the indented use though just showing that it could be done . The intention is for Prometheus to collect the metric and when collected by Prometheus it keeps a record of what pod the metrics came from. I can add an image of how it looks in Prometheus if you'd like.